### PR TITLE
i18n(fr): Add `integrations-guide/node` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `integrations-guide/node` in French language

#### Related issues & labels (optional)

- Linked to https://github.com/withastro/docs/issues/6052
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/integrations-guide/node.mdx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+// Changez ceci en fonction de votre astro.config.mjs, option `base`.
+// Ils doivent correspondre. La valeur par défaut est "/".
+const base = '/';
+app.use(base, express.static('dist/client/'));
+app.use(ssrHandler);
+
+app.listen(8080);
+```
+
+Or, with Fastify (>4):
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/integrations-guide/node.mdx:129]:
```suggestion
Ou, avec Fastify (>4) :
```

File: src/content/docs/fr/guides/integrations-guide/node.mdx [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+Dans le cas multiple exécution de variables, stockez-les dans un fichier séparé (par exemple `.env.runtime`) de `.env`. Lancez la compilation avec la commande suivante :
+
+```sh
+export $(cat .env.runtime) && astro build
+```
+
+#### Assets
+
+In En mode standalone, les ressources de votre dossier `dist/client/` sont servies par le serveur standalone. Vous pouvez déployer ces ressources sur un CDN, auquel cas le serveur ne les servira jamais. Mais dans certains cas, comme les sites intranet, il est possible de servir les ressources statiques directement depuis le serveur d'application.
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/integrations-guide/node.mdx:213]:
```suggestion
En mode standalone, les ressources de votre dossier `dist/client/` sont servies par le serveur standalone. Vous pouvez déployer ces ressources sur un CDN, auquel cas le serveur ne les servira jamais. Mais dans certains cas, comme les sites intranet, il est possible de servir les ressources statiques directement depuis le serveur d'application.
```

**Review by dreyfus92 — COMMENTED** [review]
Bon boulot @thomasbnt, voici quelques suggestions 😁

**Comment by thomasbnt** [comment]
Merci @dreyfus92 pour les suggestions ! 🚀

File: src/content/docs/fr/guides/integrations-guide/node.mdx [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+import { defineConfig } from 'astro/config';
+import node from '@astrojs/node';
+
+export default defineConfig({
+  output: 'server',
+  adapter: node({
+    mode: 'middleware',
+  }),
+});
+  ```
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/integrations-guide/node.mdx:101]:
`````suggestion
  ```js title="astro.config.mjs"
  import { defineConfig } from 'astro/config';
  import node from '@astrojs/node';
  
  export default defineConfig({
    output: 'server',
    adapter: node({
      mode: 'middleware',
    }),
  });
  ```
`````

Comment (thomasbnt) [reply src/content/docs/fr/guides/integrations-guide/node.mdx]: Thanks @Yan-Thomas !

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**